apps/client/src/app/pages/accounts/create-or-update-account-dialog/create-or-update-account-dialog.html

```diff
@@ -26,7 +30,12 @@ <h1 *ngIf="!data.account.id" i18n mat-dialog-title>Add account</h1>
     <div>
       <mat-form-field appearance="outline" class="w-100">
         <mat-label i18n>Cash Balance</mat-label>
-        <input formControlName="balance" matInput type="number" />
+        <input
+          formControlName="balance"
+          matInput
+          type="number"
+          (keydown.enter)="$event.preventDefault()"
```

Comment:
Would `(keyup.enter)="$event.stopPropagation()"` also work? Because then it would be consistent with the _Note_ textarea.

Reply:
I tried replacing `(keydown.enter)="$event.preventDefault()"` with `(keyup.enter)="$event.stopPropagation()"`. Unfortunately, it doesn't seem to work and the empty form submission occurs on pressing Enter.